python/ngrok/__main__.py

```diff
@@ -0,0 +1,216 @@
+import asyncio
```

Comment:
I just want to say that I love this devex. Feels much better than requiring additions to django manage.py, etc. 🥳

Reply:
Something to note about this approach vs the other is that with this one the process will be shown as python -m ngrok rather than the webrunner. We _could_ get around that with a reexec and some sitepackages magic however. 

We could then provide a script like so:

https://github.com/open-telemetry/opentelemetry-python-contrib/blob/a7a4f71570d7aa53be8f028155a29e2ff6a9949e/opentelemetry-instrumentation/pyproject.toml#L38

in our package

that would do something like this:

https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/opentelemetry-instrumentation/src/opentelemetry/instrumentation/auto_instrumentation/__init__.py


We could provide an `ngrok-run ...` or `ngrok-py ...` function as it is that calls the ngrok.main function to save people from putting `python -m ngrok`

Reply:
I like the runner point, I've pushed a commit to add a `ngrok-asgi` project script. Used a specific name so it doesn't block the more general ones if we want to use them in the future for other purposes.

Re-exec process magic is probably outside what I should be tackling at this time, this has already eaten more time than I anticipated. But as you mentioned in chat, if Russ is interested in the telemetry and you are too, pull requests welcome for telemetry or re-exec 😄